configs/kubernetes-rules-configmap.yaml

```diff
@@ -1095,6 +1095,8 @@ data:
         dependencies:
         - repository: apimachinery
           branch: master
+        - repository: component-base
```

Comment:
also apiserver and the different sample-* projects will depend on it.

Reply:
> will depend on it.

Is it safe to add component-base as a dependency to those repos even if it is not a dependency yet? For now, only kube-proxy, kube-scheduler and kube-controller-manager have it as a dependency (as per https://github.com/kubernetes/kubernetes/pull/72569)

Reply:
it only means that the order of publishing is accordingly. It does not change anything else.

Reply:
Great, then lets do it

Reply:
:heavy_check_mark: Done.